android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -452,12 +452,12 @@ public void setJavaScriptCanOpenWindowsAutomatically(WebView view, boolean enabl
 
   @ReactProp(name = "allowFileAccessFromFileURLs")
   public void setAllowFileAccessFromFileURLs(WebView view, boolean allow) {
-    view.getSettings().setAllowFileAccessFromFileURLs(allow);
+    view.getSettings().setAllowFileAccessFromFileURLs(false);
```

Comment:
looks like we need to keep the react properties if we are not removing e807c6f4d98c22e0db554bc8061d526d3a1bd7d5 otherwise the android build is broken.  not sure what is a better way to do this.